qp_woltka/tests/test_woltka.py

```diff
@@ -237,25 +233,20 @@ def test_woltka_to_array_rep82(self):
     def test_woltka_to_array_wol(self):
         # inserting new prep template
         prep_info_dict = {
-            'SKB8.640193': {'run_prefix': 'S22205_S104_L001_R1'},
-            'SKD8.640184': {'run_prefix': 'S22282_S102_L001_R1'}}
+            'SKB8.640193': {'run_prefix': 'S22205_S104_L001_R'},
```

Comment:
'R' vs 'R1' is an intentional change, correct?

Reply:
yup